.travis.yml

```diff
@@ -48,18 +48,38 @@ env:
     # - AWS_SECRET_ACCESS_KEY
     - secure: "F9oY34UDI8aXuZ9nb6z3/ltlqrNs4RUQHDZHa2YSFoq0r2Rn272Yg7oHfPh3deYNKxtUOBwwcpiC/7G9MqhkuLQEYhqY/KowidG6gOxCZdTDEVS10C4GDEMGj64URlUCS722tQYMIpniiB6LW2kEOOcEuzC6JsVxS7BowUEVk7Ly8BTzrgNuGFN2bSphRyrJyrS3Reo5a1YI3joUMzI29t7fegefTIHHONKhZpIE86wFRk11nwvJdt54Eu9dVFwiEiGW9L9SnG2ph4UiqN6z3iQFCQ8HNvnf+BvNasdMBR3f85DPnG/syqOcQrrE1e5T88UXRxyUwR2ty1ad+ZvQPggwsoFs1FmdJH04pGn0Tgpo/AHNe5c67tvDbw2W4jlFJgVI/keoFvQigSM2DUuL1i0v48TkZw8lgcfasRjtJi7iDTC4qqjyMLkY6y2TRTVDG/86qxK3No7vvJZajzntU3c6MUttiSDubaHL5RuklQ+7MjN5Dp1cD09L3Qqnm9KcjWzbNoyqriBO84VU4aJFF72jJ18qADX/UUj7cxi0ShY19BHX2Vrp3pPSNiePfO7okUrg5Wc4eD7u4eC+aUVHboEX35NhBnfj7/ASrsltIANYFmNarv3tvd67JIsm/Wz3pge6YOSV1pBr2IsGk5UVIXdJ/mm6GVWFlJ9PiELDBGc="
 
+git:
+  depth: false
```

Comment:
Uh whut?

Reply:
By default, travis says `git clone --depth=50` to only get the most recent 50 commits from this git history.  In order to avoid cloning everything twice though, we do `git submodule update --init --reference="$TRAVIS_BUILD_DIR"` when building the website preview.  But `--reference` isn't happy if the referenced repo is shallow.